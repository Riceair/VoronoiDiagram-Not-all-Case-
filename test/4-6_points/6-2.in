6
420 228
209 283
392 424
313 524
291 372
67 152