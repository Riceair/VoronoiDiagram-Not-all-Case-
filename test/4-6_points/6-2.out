P 67 152
P 209 283
P 291 372
P 313 524
P 392 424
P 420 228
E -234170 155318 -228.45 614.72
E -228.45 614.72 85.14 479.4
E -228.45 614.72 267.9 76.69
E -228.45 614.72 267.9 76.69
E 85.14 479.4 316.87 445.86
E 85.14 479.4 317.15 265.63
E 85.14 479.4 317.15 265.63
E 267.9 76.69 317.15 265.63
E 267.9 76.69 48123.5 -222200
E 316.87 445.86 380.46 322.35
E 316.87 445.86 63111.99 50054
E 317.15 265.63 380.46 322.35
E 380.46 322.35 123592 17924
